design/subscriber-sync.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Newsletter Subscriber synchronization
+  - As a store owner
+  - I want to update subscribers information automatically
+  - So that I can save time on my newsletter preparations
```

Comment:
I changed the wording here also. Found an article about [writing stories](https://dannorth.net/whats-in-a-story/) and there were this example wording given
> “As a [role] I want [feature] so that [benefit]” 

I think that is more clear than `As a ... I want ... In order to` ( last two other way around in monorepo ). As I think this `In order to` wording may result in blur between the feature and benefit. It can be interpreted as both.
> As a user
In order to publish information about my business
I want to create a landing page

Reply:
To me `so that` phrasing kind of removes the necessity of the feature. For example from this design document `So that I can ...` doesn't necessarily mean that the store owner will save time using this feature. Using `In order to ...` makes it so this is the goal of the feature.

Reply:
I stayed with `As .. [role] I want [feature] so that [benefit]` pattern and updated `So that I can` to more definite `So that I will`. 

One benefit from `so that` against `in order to` pattern is that you can not switch parts of the sentences. So the role->feature->benefit movement will stay the same.